dbt_subprojects/tokens/models/prices/corn/prices_corn_tokens.sql

```diff
@@ -0,0 +1,25 @@
+{% set blockchain = 'corn' %}
+
+{{ config(
+    schema = 'prices_' + blockchain,
+    alias = 'tokens',
+    materialized = 'table',
+    file_format = 'delta',
+    tags = ['static']
+    )
+}}
+
+SELECT
+    token_id
+    , '{{ blockchain }}' as blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+    ('wbtc-wrapped-bitcoin', 'wBTCN', 0xda5ddd7270381a7c2717ad10d1c0ecb19e3cdfb2, 18)
```

Comment:
Should I use wbtc price feed for wBTCN?
https://cornscan.io/token/0xda5dDd7270381A7C2717aD10D1c0ecB19e3CDFb2?chainid=21000000

Reply:
good question, let me ask 🤔

Reply:
okay, we should be good to use bitcoin pricing for this. here is answer from corn team:

> As per BTCN's pricing, it is designed to be pegged to the price of BTC indeed. It accomplishes this by introducing 1:1 backing to wBTC and cbBTC (both tightly pegged to BTC) and, hence, efficient arbitraging. BTCN's liquidity is starting to build up on Corn and hasn't met the standards for price trackers and Oracles. For this reason, assuming a 1:1 peg to BTC should be accurate enough for analytics/visualization purposes. Eventually, as it gets listed by Oracles and trackers, we should switch to its true market value.